Algorithm.CSharp/FractionalQuantityRegressionAlgorithm.cs

```diff
@@ -39,6 +40,7 @@ public override void Initialize()
 
             SetTimeZone(NodaTime.DateTimeZone.Utc);
             var security = AddSecurity(SecurityType.Crypto, "BTCUSD", Resolution.Daily, Market.GDAX, false, 3.3m, true);
+            security.BuyingPowerModel = new SecurityMarginBuyingPowerModel(3.3m);
```

Comment:
Perhaps a comment here indicating why we're doing this? Or perhaps remove using the gdax brokerage model?

Reply:
I think removing the `SetBrokerageModel` call is not enough, because now the `CashBuyingPowerModel` is the default model for `Crypto` security type.
I'll add a comment explaining why this line is required in this algorithm.

Reply:
thank you :)